modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -0,0 +1,329 @@
+
+			CustomObjectAPI.createStudentRelatedToAccount(
+				accountId = ${studentAccountId},
+				name = "Able");
+
+			ObjectDefinitionAPI.createObjectEntryWithName(
+				en_US_plural_label = "subjects",
+				name = "English");
+
+			ObjectDefinitionAPI.relateObjectEntriesByName(
+				child_plural_label = "subjects",
+				childEntryName = "English",
+				parent_plural_label = "students",
+				parentEntryName = "Able",
+				relationshipName = "studentsSubjects");
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		BatchPlanner.batchPlannerTearDown();
+
+		AccountAPI.tearDownAllAccounts();
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 4
+	test ActionsNodeAreAvailableInDonwloadedFile {
+		task ("And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("When exporting file in JSONL format with all fields selected") {
+			ImportExport.configureExport(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				exportFields = "Attribute Code",
+				exportFileFormat = "JSONL");
+
+			Button.click(button = "Export");
+
+			WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS");
+		}
+
+		task ("Then 'actions' node is available in the dowloaded file") {
+			var response = ObjectDefinitionAPI.getObjectEntries(en_US_plural_label = "students");
+
+			var entryId = JSONPathUtil.getProperty(
+				property = "$..items..[0]id",
+				response = ${response});
+
+			ImportExport.unzipDownloadedExportFile();
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
+				expectedValue = "/headless-batch-engine/${entryId}",
+				fileName = "export.jsonl",
+				jsonObject = "$.actions.delete.href");
+		}
+	}
+
+	@priority = 4
+	test AllSystemObjectFieldsAreAvailableToExport {
+		task ("When 'Account (v1_0 - Liferay Headless Admin User)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "Account (v1_0 - Liferay Headless Admin User)");
+		}
+
+		task ("Then the list of fields to select contains all the fields available with GET request except the relationship nodes") {
+			var fieldList1 = "name,accountStudents,accountUserAccounts,actions,customFields,dateCreated,dateModified";
+			var fieldList2 = "defaultBillingAddressId,defaultShippingAddressId,description,domains,externalReferenceCode,id,logoId";
+			var fieldList3 = "logoURL,numberOfUsers,organizationIds,parentAccountId,postalAddresses,status,taxId,type";
+
+			var exportFields = "${fieldList1},${fieldList2},${fieldList3}";
```

Comment:
Any reason we must combine filedList1, fieldList2 and fieldList3? And could you get all fields from response of GET requst?

Reply:
https://github.com/liferay-headless/liferay-portal/pull/1606/files#r1368225587

Reply:
Got it, it is from what Meg advised previously, thanks.